docs/markdown/Tutorials/test-custom-plugin-goal.md

```diff
@@ -0,0 +1,280 @@
+# Introduction
+
+In this tutorial, we'll learn how to test the custom plugin we wrote earlier. Pants documentation provides comprehensive coverage of the [plugin testing](https://www.pantsbuild.org/docs/rules-api-testing) and this tutorial should you to get started writing own tests.
```

Comment:
word stumbling towards the end of the sentence... ;)

Reply:
Oopsie, it was a late night :D